test/ifconfig.interfaces.test.js

```diff
@@ -192,5 +193,34 @@ describe('ifconfig', function () {
         done();
       });
     });
+
+    it('should parse the dhcp state correctly from the interfaces file', function (done) {
```

Comment:
maybe add a test that check that everything's fine when the `interfaces.FILE` does not exist and/or is not readable?

Reply:
Added a test, that checks if the rest of the interfaces gets parsed correctly, when there is an error while reading the interfaces file